cpp/core/memory/ColumnarBatch.h

```diff
@@ -77,6 +77,10 @@ class ArrowColumnarBatch final : public ColumnarBatch {
     throw gluten::GlutenException("Not implemented GetBytes for ArrowColumnarBatch");
   }
 
+  arrow::RecordBatch* GetRecordBatch() const {
```

Comment:
Is this function only used for testing? If yes, can we add some notes here?

Reply:
ok